gsoc/config.py

```diff
@@ -0,0 +1,16 @@
+import ruamel.yaml
+import os
+
+DATA_DIR = os.path.join(
+    os.path.dirname(__file__), '..',
+    '_site'
```

Comment:
Please use `STATIC_ROOT` here. (also create a newcomer issue to fix this in gci app)